portfolio/src/main/webapp/user/UserFetcher.js

```diff
@@ -0,0 +1,23 @@
+class UserFetcher {
+    constructor(fetch = window.fetch) {
+        this.fetch = fetch;
+    }
+
+    /**
+     * @returns if logged in user's email, otherwise false
```

Comment:
A function should return the same type. So if it is a string email address it should return empty string rather than false.

Reply:
Resolved, changed to empty string instead of null